tests/test_adjoint_jacobian.py

```diff
@@ -1039,3 +1077,198 @@ def casted_to_array_lightning(params):
     j_lightning = qml.jacobian(casted_to_array_lightning)(params)
 
     assert np.allclose(j_def, j_lightning)
+
+
+@pytest.mark.skipif(
+    device_name != "lightning.gpu" or not ld._CPP_BINARY_AVAILABLE,
+    reason="Tests only for lightning.gpu",
+)
```

Comment:
I think we should aim for this test to work across all devices in a follow on PR